docs/UniqueIDs.rst

```diff
@@ -0,0 +1,92 @@
+Unique IDs
+==========
+
+The database generates unique IDs and they are accessible from all nodes and relationships using the ``neo_id`` method. These keys are somewhat volatile and may be reused or change throughout a database's lifetime, so they are unsafe to use within an application.
+
+To work around this, you can define which key should act as primary key on ``Neo4j::ActiveNode`` classes instead of using the internal Neo4j ids. By default, ActiveNode will generate a unique ID using ``SecureRandom::uuid``. The instance methods ``id`` and ``uuid`` will both point to this.
```

Comment:
Thanks! ;)  It doesn't surprised me too much if the wiki is out-of-date because I don't think it's been updated since v2-v3 of the `neo4j` gem.  It will be good to have it all in one place so that it will be more in our face